OutSystems/outsystems-datagrid
# ROU-10868 Updated exportToExcel to use saveAsync

This PR will update the framework method `exportToExcel` to use `book.saveAsync`.

### What was happening
* With JSZip update to version `3.10.1` the method `book.save` is not supported by Wijmo Flexgrid since they only support it until version `2.5.0`
* Some `gulp` and `eslint` dependencies were not compatible.


### What was done
*  Updated the framework method `exportToExcel` to use `book.saveAsync`.
* Updated dev dependencies


### Test Steps
1. Go to a screen containing a Data Grid instance
2. Check the resources for JSZip v3.10.1
3. Right-click on a cell 
4. Click Export to Excel
5. Checked that it continues to work as expected


### Checklist
